### ISSUE TITLE
accessibility issues with the web ui

### ISSUE THREAD

**Issue by christopherpross** [mailcow/mailcow-dockerized#2471]
Hi,
I use the current version on mailcow, the web ui renders in google chrome (newest version). I am blind and work with the Screenreader "NVDA". The problem is, that the dialogs "mordals" are not accessible in the web ui. I have a quick look through the php code for the dialogs in mailbox.php. There i saw "aria-hidden="true". This says the screenreader, that this is invisible and it handles this dialog then so as would it not there. Maybe this is the reason. I can not create mailboxes and so on, because I can not work with this dialogs. I hope you understand the problem and can fix that.

best regards,
Christophernot accessib

**Comment by andryyy** [comment]
Hi,
Thanks for your report.
I pushed a change to set aria hidden to false when a modal is shown. Not sure if that helps, but I hope so!
